lib/sys.py

```diff
@@ -24,6 +24,8 @@
 
 goversion = Version()
 maxint = MaxInt
+maxsize = maxint
+maxunicode = 0xffff
```

Comment:
@trotterdylan, I will check remaining tomorrow morning my time, a bit late for now...